shopinvader/models/shopinvader_backend.py

```diff
@@ -99,10 +90,10 @@ class ShopinvaderBackend(models.Model):
         "etc.",
     )
     user_id = fields.Many2one(
-        related="auth_api_key_id.user_id",
         readonly=True,
         help="The technical user used to process calls to the services "
         "provided by the backend",
+        comodel_name="res.users",
     )
```

Comment:
How is this user_id field populated in jwt mode ?

Reply:
I've to check where it's used... Do we really need this field on the backend?

Reply:
I hope this field is not used for any business logic. I'd move it to shopinvader_auth_api_key.

Reply:
I'll adapt the code in this way

Reply:
implemented in https://github.com/shopinvader/odoo-shopinvader/pull/1029/commits/efc3aa845653165d30e4fcad67e16fa13d421853